plugins/scaffolder-node/src/actions/executeShellCommand.ts

```diff
@@ -45,20 +51,27 @@ export async function executeShellCommand(
     command,
     args,
     options: spawnOptions,
+    logger,
     logStream = new PassThrough(),
   } = options;
 
   await new Promise<void>((resolve, reject) => {
     const process = spawn(command, args, spawnOptions);
 
-    process.stdout.on('data', stream => {
-      logStream.write(stream);
+    process.stdout.on('data', chunk => {
+      logStream?.write(chunk);
+      logger?.log(
+        'info',
+        Buffer.isBuffer(chunk) ? chunk.toString('utf8').trim() : chunk.trim(),
+      );
     });
-
-    process.stderr.on('data', stream => {
-      logStream.write(stream);
+    process.stderr.on('data', chunk => {
+      logStream?.write(chunk);
+      logger?.log(
```

Comment:
`logger.error` here too :pray:

Reply:
All done! please let me know if this is better!